### PR TITLE
Fixed progress-path background color for dark mode

### PR DESCRIPTION
**Pull request by ajerez**
On dark mode the remaining progress bar color was the same color as the background.

### CODE REVIEW

**Comment by Akryum** [comment]
I actually like the 4th one the most 😸

**Comment by ajerez** [comment]
The 4th is the best, yeah

**Comment by ajerez** [comment]
I've updated the PR with your sugestions 🙂

File: packages/@vue/cli-ui-addon-webpack/src/components/BuildProgress.vue [hunk 1]
```diff
@@ -166,7 +166,7 @@ export default {
       width 100%
       height @width
       border-radius 50%
-      background rgba(black, .1)
+      background $vue-ui-color-dark
```
Comment (sodatea) [inline packages/@vue/cli-ui-addon-webpack/src/components/BuildProgress.vue]: This changes the color in non-dark mode too.
Should be
```suggestion
      background rgba(black, .1)
      .vue-ui-dark-mode &
        background: $vue-ui-color-dark
```

**Comment by ajerez** [comment]
PR updated. Sorry for the delay

**Pull request merged**